Homework №3 from 21.09.2022/task1.c

```diff
@@ -0,0 +1,126 @@
+#include <stdio.h>
+#include <stdlib.h>
+#define lenOfTestArrs 9
+
+void printArr(int *array, int lenOfArray)
+{
+    for (int i = 0; i < lenOfArray; ++i)
+    {
+        printf("%d ", array[i]);
+    }
+    printf("\n");
+}
+
+void insertionSort(int *arr, int arrStart, int arrEnd)
+{
+    int separator = arrStart;
+    for (int i = arrStart + 1; i <= arrEnd; ++i)
+    {
+        separator = arr[i];
+        int j = i - 1;
+        while ((arr[j] > separator) && (j >= 0))
```

Comment:
```suggestion
        while (arr[j] > separator && j >= 0)
```
В C стандартные операции имеют приоритеты по аналогии со сложением и умножением. И операции сравнения приоритетнее, чем конъюнкция. Поэтому скобки избыточны.